### PR TITLE
Add API docs for the package details template resource

### PR DESCRIPTION
**Pull request by joelverhagen**

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[497b621](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1363/commits/497b621323c89d747315870fc6fd5d0abce792a1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1363) |
[docs/api/package-details-template-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/package-details-template-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903252326501248-1363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=S1gnkxN4pv3dS8IU66hEsObw0IKQmMVn57CheiWHThc%3D&st=2019-03-25T23%3A23%3A01Z&se=2019-04-25T23%3A28%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[60f638d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1363/commits/60f638d40df15e2e3d68b98a93d23d7503e62e4a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1363) |
[docs/api/package-details-template-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/package-details-template-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C26%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903261957573731-1363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yNBQKLzoYrYykdtd42QThLg9oWrtC5P6OnzsptTulqE%3D&st=2019-03-26T19%3A54%3A24Z&se=2019-04-26T19%3A59%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by joelverhagen** [comment]
@karann-msft, this can be merged now.

File: docs/TOC.md [hunk 1]
```diff
@@ -229,9 +229,10 @@
 ## [Autocomplete](api/search-autocomplete-service-resource.md)
 ## [Package metadata](api/registration-base-url-resource.md)
 ## [Package content](api/package-base-address-resource.md)
-## [Report abuse URL](api/report-abuse-resource.md)
 ## [Repository signatures](api/repository-signatures-resource.md)
 ## [Catalog](api/catalog-resource.md)
+## [Report abuse URL](api/report-abuse-resource.md)
```
Comment (karann-msft) [inline docs/TOC.md]: We should create a new node called "Resources" and make all the resource docs as a child in alphabetical order.
Update [this doc](https://docs.microsoft.com/en-us/nuget/api/overview#resources-and-schema) to list the resources in alphabetical order too and potentially link to the resource specific page from this table.

File: docs/api/package-details-template-resource.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ms.author: jver
+ms.date: 3/1/2019
+ms.topic: reference
+ms.reviewer: ananguar
+---
+
+# Package details URL template
+
+It is possible for a client to build a URL that can be used by the user to see more package details in their web
+browser. This is useful when a package source wants show additional information about a package that may not fit within
```
Comment (karann-msft) [inline docs/api/package-details-template-resource.md]: ...wants **to** show...

File: docs/TOC.md (hunk 1 above)
Comment (joelverhagen) [reply docs/TOC.md]: Spoke offline.

- Remove code marker around resource names in overview table
- Make header names code on overview and track issue
- Make the Resources folder right after Overview

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[5a4e857](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1363/commits/5a4e8571a9dd7e43a35b7dcc1ce50e154ad97308)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1363) |
[docs/api/package-details-template-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/package-details-template-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903291733298595-1363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Q0Lbb8JeBfOQBYzE8bzfXmvdWxt0rpQio%2F0SmGte5iw%3D&st=2019-03-29T17%3A29%3A50Z&se=2019-04-29T17%3A34%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[7d3408c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1363/commits/7d3408cd1e7c34b728189d955dc3992a3c141b10)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1363) |
[docs/api/package-details-template-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-pdurl/docs/api/package-details-template-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/package-details-template-resource?branch=pr-en-us-1363) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C29%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201903291739106262-1363%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ByoAW97tlQUCv4oadpLpSEZT6%2BcZS8QlumaUTy198L8%3D&st=2019-03-29T17%3A35%3A28Z&se=2019-04-29T17%3A40%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**